### DISCUSSION TITLE
LL-HLS stream name

### DISCUSSION THREAD

**Discussion by krystian1987** [OvenMediaLabs/OvenMediaEngine#1353] [Help]
I'm trying to configure OvenMediaEngine to handle stream from LL-HLS. Everything worked fine when I sent via UDP as described in the documentation, but the name that the stream receives ("stream_${Port}") does not suit me very well... Is it possible to give the name from the value provided, for example, in the UDP metadata? Something like "stream_${EventID}", where EventId as a GUID, would be the ID of the event that started the stream?

Thank you for your help

**Comment by basisbit** [comment]
What does your configuration look like and what endpoint are you referring to? I am not sure if I correctly understood what you want to do.
OvenMediaEngine supports LLHLS output, not input.

Supported ingest/input protocols:
- Push: WebRTC, WHIP, SRT, RTMP, MPEG-2 TS
- Pull: RTSP

**Comment by dimiden** [comment]
@krystian1987 
If you are using MPEG-TS, the only macro you can use in the stream name is ```Port```. Obtaining TS metadata values to compose the stream name is a good idea and, in my opinion, it would be beneficial if applied to OME.
However, even if it's added to our backlog, its priority might be quite low.
